README.md

```diff
@@ -24,6 +25,63 @@ npm run build
 npm run test
 ```
 
+## Live testnet testing
+
+1. Check if the contract deployments repository supports the testnet you will be using. Supported chains can be found [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/info/testnet.json). If the testnet is not already supported, proceed to steps 2-4, otherwise you may skip to step 5.
+2. Navigate to the contract deployments repo [here](https://github.com/axelarnetwork/axelar-contract-deployments/) and clone the repository locally.
+3. Within the info folder in the [testnet](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/info/testnet.json) file, add the configuration of the new testnet as shown in the example below with correct values:
+
+```json
+{
+    "chains": {
+        "example": {
+            "name": "Example",
+            "id": "example",
+            "chainId": 123,
+            "rpc": "PROVIDER_RPC",
+            "tokenSymbol": "EXM",
+            "gasOptions": {
+                "gasLimit": 8000000
+            }
+        }
+    }
+}
+```
+
+4. In the root directory of this repository navigate to the `hardhat.config.js` file and modify the chains import line as shown below:
+
+```javascript
+const chains = require(`@axelar-network/axelar-contract-deployments/info/${env}.json`);
+const chains = require(`[LOCAL_PATH]/axelar-contract-deployments/info/${env}.json`);
+```
+
+5. In the root folder of this repository create a new file named `keys.json`
+6. Within `keys.json` provide your private key for the account you will be using for testing. For some tests, such as the Axelar gateway tests, you may need to provide more than one private key. Next, you may optionally provide an API key for the testnet you will be using.
+
+At this point the `keys.json` file should resemble the example file below:
+
+```json
+{
+    "accounts": ["PRIVATE_KEY1", "PRIVATE_KEY2", "PRIVATE_KEY3"]
+}
+```
```

Comment:
do we need to add [chains](https://github.com/axelarnetwork/axelar-contract-deployments/blob/b6d46f0a92d3bdd14652183ca10955ab89b70267/evm/.example.keys.json#L2) as well?

Reply:
only needed for contract explorer verification